index.html

```diff
@@ -29,13 +29,63 @@ <h1>Lofi girl</h1>
           </div>
 
           <!-- To do list function -->
-
          <div class="row" id="form">
-
           <div class="col">
-
-           <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam magni voluptatibus doloremque illum asperiores quis nemo ipsum eaque atque amet harum, ratione ipsam incidunt labore cum ex, debitis fugit architecto.</p>
-            
+           <!-- Accordion start -->
+           <div class="accordion accordion-flush" id="accordionTodoAdd">
+            <div class="accordion-item">
+              <h2 class="accordion-header" id="flush-headingOne">
+                <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#flush-collapseOne" aria-expanded="false" aria-controls="flush-collapseOne">Add item to list</button>
+              </h2>
+              <div id="flush-collapseOne" class="accordion-collapse collapse" aria-labelledby="flush-headingOne" data-bs-parent="#accordionTodoAdd">
+                <div class="accordion-body">
+                  <!-- Task create form start -->
+                  <form class="row">
+                    <div class="col-6"><!-- Task details -->
```

Comment:
Change the class to something like "col-xs-12 col-md-6" to make it responsive for mobile version (same as line 52)

Reply:
Good catch! Fixing up.